site/scss/catalog/_examples.scss

```
@@ -422,3 +542,71 @@
+}
+
+// Mirrors .moo-catalog__showcase-preview (the same tinted, bled-to-edge
+// panel the Components gallery card uses for its own preview art) --
+// bled to the popover's own edges via its padding tokens instead of the
+// card's, since this sits in a popover body, not a card body.
+.moo-examples-footer__preview {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+  // The negative margins bleed the strip to the popover's top edge; the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)